### PR TITLE
nym and nyx

### PR DESCRIPTION
**Pull request by sunnya97**
### Adding Chains

If adding a new chain, please ensure the following:
- [ ] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified.
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined.
- [ ] Add chain to bottom of `zone_chains.json`
   - RPC and REST must not have any CORS blocking of the Osmosis domain.
   - RCP node must have WSS enabled.

### Adding Assets

If adding a new asset, please ensure the following:
- [ ] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [ ] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [ ] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2631,6 +2631,22 @@
       "osmosis_verified": false,
       "osmosis_validated": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "nyx",
+      "base_denom": "unyx",
+      "path": "",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2638]: transfer/channel-15464/unyx

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -2631,6 +2631,22 @@
...
+      "osmosis_verified": true,
+      "osmosis_validated": false,
+      "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "nyx",
+      "base_denom": "unym",
+      "path": "",
```
Comment (sunnya97) [inline osmosis-1/osmosis.zone_assets.json:2646]:
```suggestion
      "path": "transfer/channel-15464/unym",
```

File: osmosis-1/osmosis.zone_assets.json (hunk 1 above)
Comment (sunnya97) [inline osmosis-1/osmosis.zone_assets.json:2638]:
```suggestion
      "path": "transfer/channel-15464/unym",
```

File: osmosis-1/osmosis.zone_assets.json [hunk 3]
```diff
@@ -2631,6 +2631,22 @@
       "osmosis_verified": false,
       "osmosis_validated": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "nyx",
+      "base_denom": "unyx",
+      "path": "transfer/channel-15464/unym",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2638]: *
transfer/channel-15464/unyx

Comment (JeremyParish69) [reply osmosis-1/osmosis.zone_assets.json:2638]: small type, unym instead on unyx

Comment (sunnya97) [inline osmosis-1/osmosis.zone_assets.json:2638]:
```suggestion
      "path": "transfer/channel-15464/unyx",
```

**Pull request merged**